src/commands/support.ts

```diff
@@ -0,0 +1,86 @@
+import {Command} from '../../typings'
+import { MessageEmbed } from 'discord.js'
+import {config} from '../../config'
+export const supportCommands:Array<Command> = [
+    {
+        name: 'exfat',
+        description: 'Displays info on why not to use exfat',
+        execute(message, args) {
+            const exampleEmbed = new MessageEmbed()
+            .setTitle("Guiformat")
+            .setURL('http://ridgecrop.co.uk/guiformat.exe')
+            .setDescription('A useful tool for formatting SD cards over 32GB as FAT32 on Windows.')
+            message.channel.send("The exFAT drivers built into the Switch has been known to corrupt SD cards and homebrew only makes this worse. Backup everything on your SD card as soon as possible and format it to FAT32. On Windows, if your SD card is over 32GB then it will not let you select FAT32 from the built-in format tool, however you can use a tool like GUIFormat to format it.", {embed:exampleEmbed})
```

Comment:
Can we change this text to the same text that behemoth has. We'd have to reword it a bit tho to be more user friendly

Reply:
how about a suggestion @hax4dazy :harold:

Reply:
Oh I rewrote the text and DM'ed it to him. Let me put it here tho.

`The standard exFAT driver is bad and shouldn't be used.

If your PC doesn't allow you to format your card to FAT32 you can do that in hekate
Tools -> Arch bit · RCM · Touch · Partition -> Partition`

Reply:
how about "should be avoided at all cost" ?
and isnt "arch bit" actually "archive bit" ? idk though, never used it.

Reply:
Nope its actually called arch bit. Also yeah "should be avoided at all cost" does sounds better.

Reply:
> Can we change this text to the same text that behemoth has. We'd have to reword it a bit tho to be more user friendly

Updated in e964cde69e793807ca3e3c1b6bb7b83a4c3a3fe2